apps/fulfillment-e2e/src/integration/login.cy.ts

```diff
@@ -1,27 +1,81 @@
+import { OauthHandlerFragment } from '../support/page_fragments/oauth-handler.fragment';
 import { PickingListsFragment } from '../support/page_fragments/picking-lists.fragment';
+import { LoginPage } from '../support/page_objects/login.page';
 
 const pickingListsFragment = new PickingListsFragment();
+const loginPage = new LoginPage();
+const oauthHandler = new OauthHandlerFragment();
 
 describe('Login Suite', () => {
   describe('when logged out', () => {
     it('should redirect to login page', () => {
       cy.visit('/');
 
       cy.location('pathname').should('to.match', /^\/login/);
+
+      loginPage.getWrapper().should('be.visible');
+      loginPage.getWrapper().find('oryx-image[resource="logo"]');
```

Comment:
We should avoid using `.get('')` and `.find('')` in test files, let's move then in page objects as UI elements.

It will make our tests more stable and easy to maintain, because selectors will be decoupled (meaning if selector changes - we should update it only in page object and not in 10 test files)